client/src/parser/convertedModelForTesting1Activity.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
IMO this shouldn't be in the repo either necessarily

Reply:
Same here. I just deleted it.